### PR TITLE
[WIP] Added APE vault, removed Aurora chain from TVL

### PR DESCRIPTION
**Pull request by leonkho**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ribbon/evm.js exports TVL: 
                
 
 
avalanche                 8.16 M
ethereum-staking          10.95 M
ethereum                  258.44 M
staking                   10.95 M

total                    266.60 M

**Comment by llamatester** [comment]
The adapter at projects/ribbon/index.js exports TVL: 
                
 
 
avalanche                 8.16 M
ethereum-staking          10.95 M
solana                    3.26 M
ethereum                  258.44 M
staking                   10.95 M

total                    266.60 M

File: projects/ribbon/evm.js [hunk 1]
```diff
@@ -99,7 +91,5 @@ module.exports = {
   avalanche: {
     tvl: avaxTvl,
   },
-  aurora: {
-    tvl: auroraTvl,
-  },
+  tvl: sdk.util.sumChainTvls([ethTvl, avaxTvl]),
```
Comment (waynebruce0x) [inline projects/ribbon/evm.js:94]: can you remove this line? it isnt needed

**Comment by waynebruce0x** [comment]
are you a team member? just want to check removing aurora is okay

**Comment by leonkho** [comment]
> are you a team member? just want to check removing aurora is okay

Yes, I am part of Ribbon. We are not going to launch on Aurora, hence why we're removing it now.

**Comment by llamatester** [comment]
The adapter at projects/ribbon/evm.js exports TVL: 
                
 
 
ethereum-staking          10.95 M
avalanche                 8.12 M
ethereum                  258.33 M
staking                   10.95 M

total                    266.46 M

**Comment by llamatester** [comment]
The adapter at projects/ribbon/index.js exports TVL: 
                
 
 
avalanche                 8.12 M
solana                    3.26 M
ethereum-staking          10.95 M
ethereum                  258.33 M
staking                   10.95 M

total                    269.72 M

**Pull request merged**